terminus-ui/validation-messages/src/validation-messages.service.ts

```diff
@@ -98,6 +100,4 @@ export class TsValidationMessagesService {
     return config[validatorName];
   }
 
```

Comment:
I think we're invoking this in other places. I remember recently thinking it's pretty weird to to pass a datePipe just to get a validation message

Reply:
Can you clarify what you mean by this?